src/snackbar/snackbar-context.js

```diff
@@ -93,7 +93,7 @@ export default function SnackbarProvider({
     setTimeout(() => {
       setAnimating(false);
       display(duration);
-    }, 1000);
+    }, 0);
```

Comment:
hey @chasestarr thanks for looking - the timeouts are still needed with this approach. from what I understand the initial setting of `animating` prop to `true` applies the `transition` property then immediately flipping it to `false` applies the `transform` [without triggering a browser reflow](https://gist.github.com/paulirish/5d52fb081b3570c81e3a) with the correct `containerHeight` value. there are two different `containerHeight` values returned when enqueueing, the second one being the correct one calculated after those^ state updates.

to me this could be refactored to be more clear but updating this to behave like a `setImmediate` fixes the delay issue.

Reply:
👍 thanks!